### DISCUSSION TITLE
PopOver: Trigger click in unit-test does not work in Vue

### DISCUSSION THREAD

**Discussion by Flyingdot** [tailwindlabs/headlessui#2639] [Help]
I want to change the popover-state to *open* in a vue unit-test to verify the popover-panels content:

```typescript
        it('should render tenants', async () => {
            await wrapper.findComponent(PopoverButton).trigger('click')

            expect(wrapper.html()).toContain(tenant.description)
        })
```

But, if I look into `wrapper.html()`, the panel is not rendered after the click and also the `data-headlessui-state` attribute is not set to `open`.

Does anyone have a clue?

**Comment by benoitkopp** [comment]
Any solution for this ?
